scripts/tests.sh

```diff
@@ -30,7 +30,11 @@ set -e
 
```

Comment:
line 29 `set -e`, @chriseth did you really mean to *disable* error checking (-e is default), usually you do `+e` instead to hard-fail when an error code is unchecked and failing.